src/main/java/com/faforever/server/entity/MapVersion.java

```diff
@@ -56,4 +58,7 @@
   @Column(name = "update_time")
   private Timestamp updateTime;
 
+  @ManyToOne(fetch = FetchType.EAGER, optional = false)
```

Comment:
Is there a specific reason we need eager loading here? If so, skip the definition because eager is default. If not, keep it lazy.